src/layouts/UnsubscribePage.jsx

```diff
@@ -12,9 +12,24 @@ const { title: seoTitle, description: seoDescription } = seo.unsubscribePage;
 
 const useStyles = createUseStyles({
   title: {
-    fontSize: '27px',
+    fontSize: '1.5rem',
     fontWeight: '700',
     color: '#33485C',
+    marginTop: '1.04rem',
+
+    '& svg': {
+      marginLeft: '8px',
+    },
+  },
+  bodyText: {
+    fontSize: '1rem',
+    color: '#33485C',
+    lineHeight: '1.389rem',
+    margin: '20px 0 30px 0',
+  },
+  button: {
+    composes: 'btn text-uppercase btn-primary mb-5',
+    width: '11.111rem',
```

Comment:
```suggestion
    width: '13.111rem',
```
![Screenshot from 2021-12-17 17-07-49](https://user-images.githubusercontent.com/11008116/146573832-368e997b-bebc-439e-875a-b982450e9775.png)